keps/sig-cli/2590-kubectl-subresource/README.md

```diff
@@ -207,45 +210,60 @@ to implement this enhancement.
 
 ##### Unit tests
 
-- `k8s.io/kubernetes/pkg/printers/internalversion`: `2023-01-12` - 71.2
-- `k8s.io/kubernetes/vendor/k8s.io/cli-runtime/pkg/resource`: `2023-01-12` - 70.9
-- `k8s.io/kubernetes/vendor/k8s.io/kubectl/pkg/cmd/edit`: `2023-01-12` - 100
-- `k8s.io/kubernetes/vendor/k8s.io/kubectl/pkg/cmd/get`: `2023-01-12` - 80.7
-- `k8s.io/kubernetes/vendor/k8s.io/kubectl/pkg/cmd/patch`: `2023-01-12` - 56.3
-- `k8s.io/kubernetes/vendor/k8s.io/kubectl/pkg/cmd/replace`: `2023-01-12` - 63.6
+- `k8s.io/kubernetes/pkg/printers/internalversion`: `2025-01-23` - 73.5
+- `k8s.io/cli-runtime/pkg/resource`: `2025-01-23` - 71.8
+- `k8s.io/kubectl/pkg/cmd/apply`: `2025-01-23` - 82
+- `k8s.io/kubectl/pkg/cmd/edit`: `2025-01-23` - 100
+- `k8s.io/kubectl/pkg/cmd/get`: `2025-01-23` - 80.8
+- `k8s.io/kubectl/pkg/cmd/patch`: `2025-01-23` - 56.4
+- `k8s.io/kubectl/pkg/cmd/replace`: `2025-01-23` - 63.8
 
 ##### Integration tests
 
-- `kubectl get`: [link to test coverage](https://github.com/kubernetes/kubernetes/blob/4802d7bb62c2623be8e4f940f6b5c1fcddd6c744/test/cmd/get.sh#L178-L184)
+- [kubectl get](https://github.com/kubernetes/kubernetes/blob/00fa8f119077da3c96090aa5efc5dfc9c5a78977/test/cmd/get.sh#L178-L184): https://storage.googleapis.com/k8s-triage/index.html?pr=1&job=pull-kubernetes-integration&test=test-cmd%3A%20run_kubectl_get_tests
+- [kubectl apply](https://github.com/kubernetes/kubernetes/blob/00fa8f119077da3c96090aa5efc5dfc9c5a78977/test/cmd/apply.sh#L417): https://storage.googleapis.com/k8s-triage/index.html?pr=1&job=pull-kubernetes-integration&test=test-cmd%3A%20run_kubectl_server_side_apply_tests
+- [TestGetSubresourcesAsTables](https://github.com/kubernetes/kubernetes/blob/00fa8f119077da3c96090aa5efc5dfc9c5a78977/test/integration/apiserver/apiserver_test.go#L1458-L1678): https://storage.googleapis.com/k8s-triage/index.html?pr=1&job=pull-kubernetes-integration&test=TestGetSubresourcesAsTables
+
+TODO:
+
+- Add a test ensuring ALL scale sub-resources continue to support tables.
```

Comment:
@nikhita this is the test you've mentioned, and the PR for it is in https://github.com/kubernetes/kubernetes/pull/129858